### PR TITLE
[13.0][FIX] stock: handle module stock_production_lot_multi_company migration

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Oversight of https://github.com/OCA/OpenUpgrade/pull/2694.

### CODE REVIEW

File: addons/stock/migrations/13.0.1.1/post-migration.py [hunk 1]
```diff
@@ -353,6 +353,13 @@ def map_stock_locations(env, main_company):
     """)
 
 
+def stock_production_lot_multi_company_migration(env):
+    rule = env.ref("stock.stock_production_lot_rule", raise_if_not_found=False)
+    if 'user' in rule.domain_force:
+        openupgrade.load_data(
+            env.cr, "stock", "migrations/13.0.1.1/noupdate_changes3.xml")
```
Comment (pedrobaeza) [inline addons/stock/migrations/13.0.1.1/post-migration.py]: Isn't better to simply write here the domain with Python?

Comment (MiquelRForgeFlow) [reply addons/stock/migrations/13.0.1.1/post-migration.py]: I thought so. But as the name also changes I thought doing by xml. I could also do a write with the two fields... but meh.

Comment (MiquelRForgeFlow) [reply addons/stock/migrations/13.0.1.1/post-migration.py]: What do you prefer?

**Review by pedrobaeza — APPROVED** [review]
To be merged when finished

**Pull request merged**